### PR TITLE
QPPSF-6546: APM validation file

### PR DESCRIPTION
**Pull request by saquino0827**
### Information
- Allows the rest api to have a configurable APM validation file. Testing environments can now allow any value we choose to be a proper APM. The validation from the Submission API can specify any test APMs they use and we can configure this to work for CT.

### Checklist 
- [ ] All JUnit tests pass (`mvn clean verify`).
- [ ] New unit tests written to cover new functionality.
- [ ] Added and updated JavaDocs for non-test classes and methods.
- [ ] No local design debt. Do you feel that something is "ugly" after your changes?
- [ ] Updated documentation (`README.md`, etc.) depending if the changes require it.

### CODE REVIEW

File: rest-api/src/test/resources/test_apm_entity_ids.json [hunk 1]
```diff
@@ -0,0 +1 @@
+["DogCow", "TestApmEntityId"]
```
Comment (KyleApfel) [inline rest-api/src/test/resources/test_apm_entity_ids.json:1]:
```suggestion
["A123", "1234-0000"]
```
Would it be better to do something that looks more like an APM entity id here?

**Review by KyleApfel — APPROVED** [review]
Looks good.

File: rest-api/src/test/resources/test_apm_entity_ids.json (hunk 1 above)
Comment (saquino0827) [reply rest-api/src/test/resources/test_apm_entity_ids.json:1]: We could, as these are just test values. The only effect would be resetting the test files to use these. I can go ahead and do that.

**Comment by saquino0827** [comment]
Admin merging until the Sonar is fixed.

**Pull request merged**